tests/scripts/thread-cert/node.py

```diff
@@ -1451,6 +1451,34 @@ def reset_trel_counters(self):
         self.send_command(cmd)
         self._expect_done()
 
+    def set_epskc(self, passcode: str, lifetime: int):
+        cmd = 'ba ephemeralkey set ' + passcode + ' ' + str(lifetime)
```

Comment:
note that lifetime is optional, and suggest to update to be more general and support optional port as well as cli usage `ba ephemeralkey set <keystring> [timeout-in-msec] [port]`

Reply:
Done